src/integration.rs

```diff
@@ -70,13 +63,13 @@ impl<A: AllocatorTrait> Integration<A> {
         swapchain: vk::SwapchainKHR,
         surface_format: vk::SurfaceFormatKHR,
     ) -> Self {
-
         // Create context
         let context = Context::default();
         context.set_fonts(font_definitions);
         context.set_style(style);
 
-        let egui_winit = egui_winit::State::new(&event_loop);
+        let mut egui_winit = egui_winit::State::new(&event_loop);
+        egui_winit.set_pixels_per_point(scale_factor as f32);
```

Comment:
^ This line was missing. Previously, on high DPI monitors, the egui-winit-ash-integration would stretch the textures too much.